### DISCUSSION TITLE
External input device - Keyboard - Key assignments - Change?

### DISCUSSION THREAD

**Discussion by hubertusw** [osmandapp/OsmAnd#20687] [Q&A] [answered]
Is there an option to change predefined key assignments or to add further ones? I'm using a Bluetooth remote control and would like to make use of all keys. My device 'Joymap 5.10' supports a mode called 'AndroidExtra' which among other feature supports short and long press of keys.

**Comment by dmpr0** [comment, marked as answer]
Hi! Yes you need to add new type in Menu – Settings – Profile – General settings – External input device – Type – Add

**Comment by hubertusw** [comment]
Got it, thanks a lot!
